nuitka/freezer/DllDependenciesPosix.py

```diff
@@ -44,32 +43,35 @@ def detectBinaryPathDLLsPosix(dll_filename, package_name, original_dir):
     # on Travis. pylint: disable=global-statement
     global _detected_python_rpath
     if _detected_python_rpath is None and not isPosixWindows():
-        _detected_python_rpath = getSharedLibraryRPATH(sys.executable) or False
+        _detected_python_rpath = getSharedLibraryRPATHS(sys.executable) or []
 
         if _detected_python_rpath:
             # Need to resolve a potential symlink.
             if os.path.islink(sys.executable):
-                sys_executable = resolveSymlink(sys.executable)
+                sys_executable = os.path.realpath(sys.executable)
```

Comment:
I think this is an existing bug that I can submit separately: resolveSymlink returns a relative path, not an absolute path, so it's not helpful here and the further filename manipulations below do the wrong thing.

Reply:
You are absolutely spot on, resolving it ourselves, makes only sense for things within rpaths, but not there. I am going to just fix it too.

Reply:
This was a conflict, since that change was already in a hotfix in the hopes it already helps something